### PR TITLE
feat: Add MeteringPointConnected event.

### PR DESCRIPTION
**Pull request by defectiveAi**
- Add ingestion of MeteringPointConnected.
- Forward MeteringPointConnected to EventHub.

### CODE REVIEW

File: source/dotnet/IntegrationEventListener/MeteringPointConnectedListenerEndpoint.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        private readonly MeteringPointConnectedDtoFactory _meteringPointConnectedDtoFactory;
+        private readonly IJsonSerializer _jsonSerializer;
+
+        public MeteringPointConnectedListenerEndpoint(
+            MeteringPointConnectedDtoFactory meteringPointConnectedDtoFactory,
+            IJsonSerializer jsonSerializer)
+        {
+            _meteringPointConnectedDtoFactory = meteringPointConnectedDtoFactory;
+            _jsonSerializer = jsonSerializer;
+        }
```
Comment (BjarkeMeier) [inline source/dotnet/IntegrationEventListener/MeteringPointConnectedListenerEndpoint.cs:36]: I _think_ you can get rid of all this boilerplate by making class and Run method static and passe these deps directly to the method.

Comment (defectiveAi) [reply source/dotnet/IntegrationEventListener/MeteringPointConnectedListenerEndpoint.cs:36]: We talked about it.

File: source/dotnet/IntegrationTests/Endpoint/MeteringPointConnectedListenerEndpointTests.cs [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+            var allReceived = whenAllEvent.Wait(TimeSpan.FromSeconds(5));
+            allReceived.Should().BeTrue();
+
+            // Only one event is expected
+            var actual = jsonSerializer.Deserialize<MeteringPointConnectedDto>(
+                Fixture.EventHubListener
+                    .ReceivedEvents.Single()
+                    .Data.ToString());
+            actual.EffectiveDate.Should().Be(effectiveDate.ToInstant());
```
[the remark this answers is not in the document]
Comment (defectiveAi) [reply source/dotnet/IntegrationTests/Endpoint/MeteringPointConnectedListenerEndpointTests.cs:88]: Good call - Lasse and I will look into it in a separate PR.

[the remark this answers is not in the document]
Comment (defectiveAi) [reply source/dotnet/IntegrationTests/Endpoint/MeteringPointConnectedListenerEndpointTests.cs]: CorrelationId is now sent as part of message. Updated UT in this PR to check the message. Also the AppInsigts graph works already.

[the remark this answers is not in the document]
Comment (defectiveAi) [reply source/dotnet/IntegrationTests/Endpoint/MeteringPointConnectedListenerEndpointTests.cs]: Nvm, it does not work like that... :(

**Pull request merged**